app/src/main/res/values-sw480dp/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we be handling tablet layouts here? I suggest avoiding that for now and deferring to whoever implements tablet support holistically (we may not even use the same exact layout for tablets).

Reply:
Removed.